robocopkaka/bootcamp-29-project
# Fix handle errors during adding centers

#### What does this PR do?
Dispatch action that handles failed attempts to add a center
#### Description of Task to be completed?
* Dispatch action that handles failed attempts to add a center
* Dispatch action used for showing a loading spinner while a request to add a center is ongoing
* Add cases to the centerReducer to handle both actions
* Show toasts to the user specifying the state of their attempt to add a center
#### How should this be manually tested?
When a user tries to add a center, they should see a toast telling them if their attempt was successful or not. If the attempt fails, they should be told why.
#### Any background context you want to provide?
Wasn't showing toasts and dispatching an action to handle failed attempts
#### What are the relevant pivotal tracker stories?(if applicable)
#155877761
